cgm/src/main/java/org/codice/imaging/cgm/CommonSpecificationModeElement.java

```diff
@@ -55,16 +55,16 @@ public void readParameters(final CgmInputReader inputReader, final int parameter
         int data = inputReader.readEnumValue();
         switch (data) {
             case LINE_WIDTH_MODE_ABSOLUTE:
-                mode = LineWidthSpecificationModeElement.Mode.ABSOLUTE;
+                mode = CommonSpecificationModeElement.Mode.ABSOLUTE;

```

Comment:
Was SonarLint unhappy about the enumeration being different than the file name?

Reply:
Sonar was upset that a static class member was being accessed from the class that extended it. 

> In the interest of code clarity, static members of a base class should never be accessed using a derived type's name. Doing so is confusing and could create the illusion that two different static members exist.

Looking back I should have shorten it to just the nested enum instead of including the class name `Mode.XXXX`. Will update